operator/config/controller/deployment.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kit-controller
+  namespace: kit
+  labels:
+    control-plane: kit-controller
+spec:
+  selector:
+    matchLabels:
+      control-plane: kit-controller
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        control-plane: kit-controller
+    spec:
+      serviceAccountName: kit-controller
+      containers:
+      - name: manager
+        image: ko://github.com/awslabs/kit/operator/cmd/controller
+        resources:
+          requests:
+            cpu: 100m
+            memory: 20Mi
+      # https://github.com/aws/amazon-eks-pod-identity-webhook/issues/8#issuecomment-636888074
+      securityContext:
+        fsGroup: 1000
+      nodeSelector:
+        kubernetes.io/hostname: ip-192-168-12-204.us-west-2.compute.internal
+      tolerations:
```

Comment:
Why this?

Reply:
Nice catch! I forgot to remove this I was using this for local testing. I will open another PR to remove this.